boltons/dictutils.pyi

```diff
@@ -0,0 +1,118 @@
+try:
+    from collections.abc import ItemsView, KeysView, ValuesView
+except ImportError:
+    from collections import KeysView, ValuesView, ItemsView
+
+from collections.abc import Callable, Generator, Hashable, Iterable, Iterator
+from typing import Any, Never
+
+class OrderedMultiDict(dict):
+    def __init__(self, *args, **kwargs) -> None: ...
+    def add(self, k: Hashable, v: Any) -> None: ...
+    def addlist(self, k: Hashable, v: Any) -> None: ...
+    def get(self, k: Hashable, default: Any | None = ...) -> Any: ...
+    def getlist(self, k: Hashable, default=...) -> list[Any]: ...
+    def clear(self) -> None: ...
```

Comment:
The same of all other methods that are the same (in terms of signatures) in superclasses.